cache/segmenting.go

```diff
@@ -11,8 +11,8 @@ type SegmentingCache struct {
 
 type StreamInfo struct {
 	SourceFile            string
-	CallbackUrl           string
-	UploadDir             string
+	CallbackURL           string
+	UploadURL             string
```

Comment:
Changed Url -> URL because that's Go convention.

Changed "Dir" to URL because it's a URL and not a Dir